explore/src/main/scala/explore/EditableLabel.scala

```diff
@@ -139,13 +141,15 @@ object EditableLabel {
         if (editing.value.value)
           <.div(^.width := "100%", ^.display.flex)(
             InputText(
-              id = "editable-label-input", // won't necesessarily be unique...
+              id = s"editable-label-input-${Random.nextInt}",
```

Comment:
I think this should be memoized. Otherwise it will change on each render.

Reply:
Actually, can we pass de id as a required property?

Then the caller can pass the `obsId` for example.